### ISSUE TITLE
Make the user pool email mutable

### ISSUE THREAD

**Issue by douglasnaphas** [douglasnaphas/madliberation#329]
Basically if you log in with one social provider, and then log in with the same email address with another social provider, AWS for some reason tries to update your email address in the user pool, and gets an error if the email address is immutable.

Discussed here:

https://forums.aws.amazon.com/thread.jspa?threadID=256704

I would need to delete the user pool and re-create it to fix this. I'm deciding to not fix this for Passover 2021, because this is too risky of a change for the day of the seder. I did, however, already delete the user pool domain and user pool in the madliberation-test and madliberation accounts, which was probably a mistake, because the warning message when I deleted the domain said something like "it may take up to an hour to re-create this domain." Oh, well. I didn't delete anything from the prod account.

**Comment by douglasnaphas** [comment]
![Screen Shot 2021-03-27 at 11 45 37 AM](https://user-images.githubusercontent.com/5433410/112727379-a1a65c00-8ef8-11eb-88c1-ad701cb67208.png)

**Comment by douglasnaphas** [comment]
https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-cognito.StandardAttribute.html#mutable

**Comment by douglasnaphas** [comment]
I'm disabling all social providers for 2021 because of this issue. I'll do so by deleting the params in param store that configure the social providers. People will still be able to log in using a username and password.

**Comment by douglasnaphas** [comment]
Social login is working now as far as we can tell after the tense redeployments.

**Comment by douglasnaphas** [comment]
One user was able to log in with Facebook and resume with Google, as a seder leader. Another was able to log in with Google and resume with Amazon, as a participant.

**Comment by douglasnaphas** [comment]
> I'm disabling all social providers for 2021 because of this issue. I'll do so by deleting the params in param store that configure the social providers. People will still be able to log in using a username and password.

After some reconsideration of the above, I went ahead with https://github.com/douglasnaphas/madliberation/commit/a35bd72b561127273a48c40027f2cbdacb33003d, in which I deleted the user pool. Then, with the commits in the vicinity of that commit, following it, I re-deployed with `mutable: true`. It worked fine, and the whole process took like 25 minutes.
